### ISSUE TITLE
new lint: passing 0 as length to Vec::resize

### ISSUE THREAD

**Issue by xfix** [rust-lang/rust-clippy#5444]
Clippy should inform about passing `0` as length to resize. It's not useful, and it's likely a mistake caused by swapping the parameter order.

Example code:

```rust
fn main() {
    let mut v = Vec::new();
    v.resize(0, 1024);
}
```

**Comment by JarredAllen** [comment]
I can implement this.

**Comment by mlegner** [comment]
Are you still working on this, @JarredAllen? Otherwise, I could have a go at it.

**Comment by JarredAllen** [comment]
I had said I would do this thinking that the other issue I was working on would wrap up quickly, but then it turned out that there was more to do (and I now also have to deal with finals). If you want to handle this, feel free to do so.

**Comment by montrivo** [comment]
@mlegner are you still active on this one?

**Comment by mlegner** [comment]
@montrivo I'm sorry, I completely forgot about this issue. :disappointed: 
No, I'm currently not working on it and won't have time in the next few days. So, if you're interested, go ahead.

**Comment by montrivo** [comment]
Thanks for the quick answer. I'll take a look at it.
